Source/ACE.Server/WorldObjects/Creature_Death.cs

```diff
@@ -200,19 +200,17 @@ protected void CreateCorpse(WorldObject killer)
 
             //corpse.Location.PositionZ = corpse.Location.PositionZ - .5f; // Adding BaseDescriptionFlags |= ObjectDescriptionFlag.Corpse to Corpse objects made them immune to gravity.. this seems to fix floating corpse...
 
-            corpse.OwnerId = Guid.Full;
+            corpse.VictimId = Guid.Full;
             corpse.Name = $"Corpse of {Name}";
 
-            LandblockManager.AddObject(corpse);
```

Comment:
To be clear here, this is a duplicate AddObject line.. there is another at the end of the func and should be the one left